openrtb_ext/imp_lockerdome.go

```diff
@@ -0,0 +1,7 @@
+package openrtb_ext
+
```

Comment:
This ext struct `ExtImpLockerDome` isn't needed as you're forwarding the request as is and not trying to unmarshal the ext at any point before calling your API